games-kids/tuxmath/tuxmath-1.7.2.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="Tux4Kids' math tutor for kids"
+DESCRIPTION="TuxMath is an educational math tutor for kids starring Tux, \
+the Linux penguin. This educational game comes with two different games for \
+practicing math, and having a great time doing it."
+HOMEPAGE="https://tux4kids.alioth.debian.org/"
+COPYRIGHT="2000-2009 Sam Hart"
+LICENSE="GNU GPL v2"
+REVISION="2"
+SOURCE_URI="https://sf.net/projects/tuxmath/files/tuxmath-source/TuxMath%201.7.2%20-%20Source/tuxmath_w_fonts-1.7.2.tar.gz"
```

Comment:
Always strikes me that you can find a "shorter" download link, I usualy use the one provided by the download page, thanks!